modules/affile/wildcard-source.c

```diff
@@ -208,6 +215,13 @@ _add_directory_monitor(WildcardSourceDriver *self, const gchar *directory)
     .method = self->monitor_method
   };
   DirectoryMonitor *monitor = create_directory_monitor(&options);
+  if (!monitor)
```

Comment:
The `_add_directory_monitor` is also called from the `_init`, thus the wildcard file source could be initialized even if there is no `monitor` object exits, thus does nothing.

Reply:
Thanks for comment! Unfortunately I do not know how trigger, but I added a new patch that should resolve it.